### PR TITLE
translate definition file in english

### LINKED ISSUE #1135: doc tooltips: The text is always in french
The documentation tooltip is always in french, even if the application is loaded in english.

Capture d’écran du 2024-12-06 13-04-56

### PR DESCRIPTION
**Pull request by kaligrafy**
closes #1135

### CODE REVIEW

File: packages/chaire-lib-backend/src/config/server.config.ts [hunk 1]
```diff
@@ -370,7 +370,9 @@ export const setProjectConfiguration = (newConfig: Partial<ProjectConfiguration<
                             engines: {
                                 osrmRouting: {
                                     port: osrmModesInDeprecatedFormat[osrmMode].port,
-                                    host: osrmModesInDeprecatedFormat[osrmMode].host || osrmModesInDeprecatedFormat[osrmMode].osrmPath,
+                                    host:
+                                        osrmModesInDeprecatedFormat[osrmMode].host ||
+                                        osrmModesInDeprecatedFormat[osrmMode].osrmPath,
```
Comment (greenscientist) [inline packages/chaire-lib-backend/src/config/server.config.ts:375]: THis is not related to this PR

File: packages/transition-backend/file/definitions/definitions-en.tex [hunk 2]
```diff
@@ -605,782 +587,771 @@ \subsection*{Temps et durées • \textit{Time and durations}}
...
-\makecell[r]{Réseau \\ \textit{Network}} & \[N^T\] & - & - & Réseau de transport collectif comprenant des lignes et des services/horaires pour une ou plusieurs agences \\
+\makecell[r]{Network} & \[N^T\] & - & - & Public transit network including lines and services/schedules for one or more agencies \\
 \hline
 \label{service}
-\makecell[r]{Service \\ \textit{Service}} & \[H\] &  &  & Ensemble d'horaires associés pour une période ou un type de service donné (service de semaine, de fin de semaine, de nuit, service spécial, etc.). \\
+\makecell[r]{Service} & \[H\] &  &  & Set of associated schedules for a given period or service type (weekday service, weekend service, night service, special service, etc.) \\
 \hline
 \label{scenario}
-\makecell[r]{Scénario \\ \textit{Scenario}} & \[H_S\] &  &  & Ensemble de services utilisé pour des fins de design, de comparaison et de simulation. \\
+\makecell[r]{cenario} & \[H_S\] &  &  & Set of services used for design, comparison and simulation purposes \\
```
Comment (greenscientist) [inline packages/transition-backend/file/definitions/definitions-en.tex:1231]: Missing S to cenario here

File: packages/transition-backend/file/definitions/definitions-en.tex [hunk 3]
```diff
@@ -102,73 +109,72 @@ \subsection*{Stops, stop nodes, stations and segments}
...
+\makecell[r]{Depot} & [G] & - & - & Bus garage or rail depot where units and vehicles are stored and where maintenance is performed. Usually, the depot is also the departure and arrival point for vehicles and drivers assigned to service at the beginning and end of their runs. \
 \hline
 \label{depot_unit_capacity}
-\makecell[r]{Capacité d'unités du garage \\ \textit{Depot units capacity}} & \[{N_u}_G\] & unités & - & Nombre d'unités pouvant être stationnées simultanément dans le garage ou le dépôt. Attention de bien spécifier si cette capacité comprend les emplacements réservés aux véhicules en maintenance. \\
+\makecell[r]{Depot units capacity} & [{N_u}G] & units & - & Number of units that can be simultaneously parked in the garage or depot. Care should be taken to specify whether this capacity includes spaces reserved for vehicles under maintenance and/or being charged. \
```
Comment (greenscientist) [inline packages/transition-backend/file/definitions/definitions-en.tex:135]: The underscore before the G got dropped
\[{N_u}_G\]  -> [{N_u}G]

**Review by greenscientist — CHANGES_REQUESTED** [review]

**Review by greenscientist — COMMENTED** [review]
You should add the test as a separate commit

**Review by samuel-duhaime — COMMENTED** [review]
Je n'ai rien à ajouter par rapport aux autres pour ce review.

File: packages/transition-backend/file/__tests__/definitions.test.ts [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+    test('both files have the same number of labels', () => {
+        expect(enLabels.size).toBe(frLabels.size);
+    });
+
+    test('labels appear in the same order in both files', () => {
+        const enLabelArray = [...enLabels];
+        const frLabelArray = [...frLabels];
+
+        expect(enLabelArray).toEqual(frLabelArray);
+    });
```
[the remark this answers is not in the document]
Comment (kaligrafy) [reply packages/transition-backend/file/__tests__/definitions.test.ts]: removed test, moved to another PR

**Comment by kaligrafy** [comment]
> You should add the test as a separate commit

done!

**Review by GabrielBruno24 — APPROVED** [review]

File: locales/fr/notifications.json [hunk 5]
```diff
@@ -56,7 +56,7 @@
   "CalculateRouting": "Calculs de chemin",
   "CalculateOdRouting": "Chargement OD",
   "CalculateAccessibilityMap": "Calcul du polygone d'accessibilité",
-  "UpdatingPathRoute": "Mise à jour du parcours du trajet",
```
Comment (greenscientist) [inline locales/fr/notifications.json:59]: I feel like this change should be in the other commit about parcours....

File: packages/transition-backend/file/definitions/definitions-en.tex [hunk 6]
```diff
@@ -102,73 +109,73 @@ \subsection*{Stops, stop nodes, stations and segments}
...
+}
 \hline
-\makecell[r]{Définition \\ \textit{Definition}} & \makecell[c]{Symbole \\ \textit{Symbol}} & \makecell[c]{Unité \\ \textit{Unit}} & \makecell[c]{Expression \\ \textit{Expression}} & \makecell[l]{Description \\ \textit{Description}} \\ 
+\makecell[r]{Definition} & \makecell[c]{Symbol} & \makecell[c]{Unit} & \makecell[c]{Expression} & \makecell[l]{Description} \\
 \hline
 \hline
 \endhead
 \label{transit_unit}
-\makecell[r]{Unité de transport collectif \\ \textit{Transit unit (Unit)}} & \[u\] & - & - & Ensemble de voitures, de wagons ou de remorques couplés constituant un train. Un bus régulier ou articulé représente une seule unité. \\
+\makecell[r]{Transit unit} & \[u\] & - & - & Set of cars, wagons, or trailers coupled together forming a train. A regular or articulated bus represents a single unit. \\
```
Comment (greenscientist) [inline packages/transition-backend/file/definitions/definitions-en.tex:127]: Just making sure that you wanted to drop the "(unit)" here

File: packages/transition-backend/file/definitions/definitions-en.tex [hunk 7]
```diff
@@ -102,73 +109,73 @@ \subsection*{Stops, stop nodes, stations and segments}
...
-\makecell[r]{Taille de la flotte en maintenance \\ \textit{Maintenance fleet size}} & \[F_m\] & unités & - & Nombre d'unités en réparation ou en maintenance préventive. Ces unités ne peuvent être utilisées pour effectuer du service. \\
+\makecell[r]{Maintenance fleet size} & \[F_m\] & units & - & Number of units under repair or preventive maintenance. These units cannot be used for service. \\
 \hline
 \label{service_fleet_size}
-\makecell[r]{Taille de la flotte en service \\ \textit{Service fleet size}} & \[F_s\] & unités & \[F - F_m - F_r\] & Nombre d'unités pouvant être utilisées pour effectuer le service planifié pendant la période de pointe maximale. \\
+\makecell[r]{Service fleet size} & \[F_s\] & units & \[F - F_m - F_r\] & Number of units that can be used to perform planned service during maximum peak period. \\
 \hline
 \label{fleet_use_ratio}
-\makecell[r]{Ratio d'utilisation de la flotte \\ \textit{Fleet use ratio}} & \[\mu_{F_u}\] & - & \[\frac{F_s + F_r}{F}\] & - \\
+\makecell[r]{Fleet use ratio} & \[\mu{F_u}\] & - & \[\frac{F_s + F_r}{F}\] & - \\
```
Comment (greenscientist) [inline packages/transition-backend/file/definitions/definitions-en.tex:151]: You dropped the underscopre after \mu_, check if it's voluntary

File: packages/transition-backend/file/definitions/definitions-en.tex [hunk 8]
```diff
@@ -178,118 +185,100 @@ \subsection*{Véhicules, unités et garages • \textit{Vehicles, units and depo
...
-\label{unit_seated_capacity}
-\makecell[r]{Capacité de sièges par unité \\ \textit{Unit seated capacity}} & \[C_{u_{se}}\] & places & \[n_y C_{{y}_{se}}\] si toutes les voitures de l'unité ont la même capacité de sièges. & - \\
-\hline
-\label{unit_standees_capacity}
-\makecell[r]{Capacité de passagers debout par unité \\ \textit{Unit standees capacity}} & \[C_{u_{st}}\] & places & \[n_y C_{{y}_{st}}\] si toutes les voitures de l'unité ont la même capacité de places debout. & - \\
+\makecell[r]{Floor area per standee} & \[e\] & \[m^2\] & - & Floor area allocated for each standing place (footprint). Comfortable, adequate circulation: \(0.25 m^2\); Uncomfortable, difficult circulation: \(0.15 m^2\) \\
 \hline
 \label{vehicle_boarding_door_channels}
-\makecell[r]{Nombre de voies d'embarquement par véhicule \\ \textit{Number of boarding door channels per vehicle}} & \[{n_{Bch}}_y\] & voies d'embarquement & - & Nombre de voies permettant l'embarquement d'une personne à la fois dans le véhicule. \\
+\makecell[r]{Number of boarding door channels per vehicle} & \[{n_{Bch}}y\] &boarding channels & - & Number of channels allowing one person at a time to board the vehicle. \\
```
Comment (greenscientist) [inline packages/transition-backend/file/definitions/definitions-en.tex]: DO you need a space after the "&" ?
Also dropped the _ before the y

File: packages/transition-backend/file/definitions/definitions-en.tex [hunk 9]
```diff
@@ -178,118 +185,100 @@ \subsection*{Véhicules, unités et garages • \textit{Vehicles, units and depo
...
 \hline
 \label{vehicle_mixed_door_channels}
-\makecell[r]{Nombre de voies d'accès mixtes par véhicule \\ \textit{Number of mixed door channels per vehicle}} & \[{n_{ABch}}_{y}\] & voies mixtes & - & Nombre de voies permettant l'embarquement ou le débarquement d'une personne à la fois dans le véhicule. \\
+\makecell[r]{Number of mixed door channels per vehicle} & \[{n{ABch}}{y}\] &mixed channels & - & Number of channels allowing one person at a time to either board or alight from the vehicle. \\
 \hline
 \label{vehicle_door_channels}
-\makecell[r]{Nombre total de voies d'accès par véhicule \\ \textit{Number of door channels per vehicle}} & \[{n_{ch}}_y\] & voies d'accès & \[{n_{Ach}}_{y} + {n_{Bch}}_{y} + {n_{ABch}}_{y}\] & Nombre de voies d'accès permettant l'embarquement, le débarquement ou les deux dans le véhicule. \\
+\makecell[r]{Number of door channels per vehicle} & \[{n{ch}}y\] & accesschannels & \[{n{Ach}}{y} + {n{Bch}}{y} + {n{ABch}}{y}\] & Number of access channels allowing boarding, alighting, or both in the vehicle. \\
```
Comment (greenscientist) [inline packages/transition-backend/file/definitions/definitions-en.tex:207]: missing _ after the n

File: packages/transition-backend/file/definitions/definitions-en.tex [hunk 10]
```diff
@@ -178,118 +185,100 @@ \subsection*{Véhicules, unités et garages • \textit{Vehicles, units and depo
...
 \hline
 \label{unit_boarding_door_channels}
-\makecell[r]{Nombre de voies d'embarquement par unité \\ \textit{Number of boarding door channels per unit}} & \[{n_{Bch}}_{u}\] & voies d'embarquement & - & Nombre de voies permettant l'embarquement d'une personne à la fois dans l'ensemble des véhicules de l'unité. \\
+\makecell[r]{Number of boarding door channels per unit} & \[{n{Bch}}{u}\] &boarding channels & - & Number of channels allowing one person at a time to board across all vehicles in the unit. \\
```
Comment (greenscientist) [inline packages/transition-backend/file/definitions/definitions-en.tex]: missing _ after the n

File: packages/transition-backend/file/definitions/definitions-en.tex [hunk 11]
```diff
@@ -178,118 +185,100 @@ \subsection*{Véhicules, unités et garages • \textit{Vehicles, units and depo
...
 \hline
 \label{unit_alighting_door_channels}
-\makecell[r]{Nombre de voies de débarquement par unité \\ \textit{Number of alighting door channels per unit}} & \[{n_{Ach}}_{u}\] & voies de débarquement & - & Nombre de voies permettant le débarquement d'une personne à la fois dans l'ensemble des véhicules de l'unité. \\
+\makecell[r]{Number of alighting door channels per unit} & \[{n{Ach}}{u}\] &alighting channels & - & Number of channels allowing one person at a time to alight across all vehicles in the unit. \\
```
Comment (greenscientist) [inline packages/transition-backend/file/definitions/definitions-en.tex]: missing _ after n, and before {u}

File: packages/transition-backend/file/definitions/definitions-en.tex [hunk 12]
```diff
@@ -178,118 +185,100 @@ \subsection*{Véhicules, unités et garages • \textit{Vehicles, units and depo
...
 \hline
 \label{unit_mixed_door_channels}
-\makecell[r]{Nombre de voies d'accès mixtes par unité \\ \textit{Number of mixed door channels per unit}} & \[{n_{ABch}}_{u}\] & voies mixtes & - & Nombre de voies permettant l'embarquement ou le débarquement d'une personne à la fois dans l'ensemble des véhicules de l'unité. \\
+\makecell[r]{Number of mixed door channels per unit} & \[{n{ABch}}{u}\] &mixed channels & - & Number of channels allowing one person at a time to either board or alight across all vehicles in the unit. \\
 \hline
 \label{unit_door_channels}
-\makecell[r]{Nombre total de voies d'accès par unité \\ \textit{Number of door channels per unit}} & \[{n_{ch}}_{u}\] & voies d'accès & \[{n_{Ach}}_{u} + {n_{Bch}}_{u} + {n_{ABch}}_{u}\] & Nombre de voies d'accès permettant l'embarquement, le débarquement ou les deux dans l'ensemble des véhicules de l'unité. \\
+\makecell[r]{Number of door channels per unit} & \[{n{ch}}{u}\] & accesschannels & \[{n{Ach}}{u} + {n{Bch}}{u} + {n{ABch}}_{u}\] & Number of access channels allowing boarding, alighting, or both across all vehicles in theunit. \\
```
Comment (greenscientist) [inline packages/transition-backend/file/definitions/definitions-en.tex:219]: Mising several underscore

**Review by greenscientist — COMMENTED** [review]
first pass

File: packages/transition-backend/file/definitions/definitions-en.tex [hunk 13]
```diff
@@ -421,178 +408,176 @@ \subsection*{Distances et longueurs • \textit{Distances and lengths}}
...
-\makecell[r]{Temps d'arrêt moyen \\ \textit{Average dwell time}} & \[\overline{t_q}\] & \[s\] & \[\frac{\sum_{i=1}^{n_q} {t_q}_i}{n_q}\] & Moyenne des temps d'arrêt d'un parcours. \\
+\makecell[r]{Average dwell time} & \[\overline{t_q}\] & \[s\] & \[\frac{\sum_{i=1}^{n_q} {t_q}i}{n_q}\] & Average of dwell times on a path. \\
 \hline
 \label{median_dwell_time}
-\makecell[r]{Temps d'arrêt median \\ \textit{Median dwell time}} & \[\widetilde{t_q}\] & \[s\] & & Médiane des temps d'arrêt d'un parcours. \\
+\makecell[r]{Median dwell time} & \[\widetilde{t_q}\] & \[s\] & & Median of dwell times on a path. \\
 \hline
 \label{stop_to_stop_time}
-\makecell[r]{Temps arrêt à arrêt \\ \textit{Stop to stop time}} & \[t_{\Delta s}\] & \[s\] & \[t_q + t_l\] & Temps total entre le départ de l'arrêt précédent et le départ de l'arrêt suivant. Comprend le temps de parcours inter-arrêts et le temps d'arrêt. \\
+\makecell[r]{Stop-to-stop time} & \[t{\Delta s}\] & \[s\] & \[t_q + t_l\] & Total time between departure from the previous stop and departure from the next stop. Includes interstop travel time and dwell time. \\
```
Comment (greenscientist) [inline packages/transition-backend/file/definitions/definitions-en.tex:511]: Missing _ after t

File: packages/transition-backend/file/definitions/definitions-en.tex [hunk 14]
```diff
@@ -605,782 +590,771 @@ \subsection*{Temps et durées • \textit{Time and durations}}
...
+\makecell[r]{Half-cycle speed} & \[V_{1/2c}\] & \[{km}/h\] & \[\frac{d_p}{T_{1/2c}}\] & Average speed during a half-cycle (path) \\
 \hline
 \label{outbound_half_cycle_speed}
-\makecell[r]{Vitesse de demi-cycle aller \\ \textit{Outbound half-cycle speed}} & \[{V_{1/2c}}^{\prime}\] & \[{km}/h\] & \[\frac{d^{\prime}}{{T_{1/2c}}^{\prime}}\] & Vitesse moyenne pendant le demi-temps de cycle aller. \\
+\makecell[r]{Outbound half-cycle speed} & \[{V_{1/2c}}^{\prime}\] & \[{km}/h\] & \[\frac{d^{\prime}}{{T_{1/2c}}^{\prime}}\] & Average speed during outbound half-cycle time. \\
 \hline
 \label{inbound_half_cycle_speed}
-\makecell[r]{Vitesse de demi-cycle retour \\ \textit{Inbound half-cycle speed}} & \[{V_{1/2c}}^{\prime\prime}\] & \[{km}/h\] & \[\frac{d^{\prime\prime}}{{T_{1/2c}}^{\prime\prime}}\] & Vitesse moyenne pendant le demi-temps de cycle retour. \\
+\makecell[r]{Inbound half-cycle speed} & \[{V_{1/2c}}^{\prime\prime}\] & \[{km}/h\] & \[\frac{d^{\prime\prime}}{{T_{1/2c}}{\prime\prime}}\] & Average speed during inbound half-cycle time. \\
```
Comment (greenscientist) [inline packages/transition-backend/file/definitions/definitions-en.tex:750]: prime prime

File: packages/transition-backend/file/definitions/definitions-en.tex [hunk 15]
```diff
@@ -605,782 +590,771 @@ \subsection*{Temps et durées • \textit{Time and durations}}
...
-\makecell[r]{Intervalle maximum \\ \textit{Maximum headway}} & \[h_{max}\] & \[min\] & \[\frac{60}{f_{min}}\] & Intervalle de service maximum selon contraintes et réglementations nationales, régionales ou locales. Cet intervalle peut être différente selon la période de service (matin, pointe du matin, journée, pointe du soir, soir, nuit, fin de semaine, etc.) \\
+\makecell[r]{Maximum headway} & \[h_{max}\] & \[min\] & \[\frac{60}{f_{min}}\] & Maximum service headway according to national, regional, or local constraints and regulations. This headway can differ depending on the service period (morning, morning peak, daytime, evening peak, evening, night, weekend, etc.) \\
 \hline
 \label{maximum_peak_headway}
-\makecell[r]{Intervalle maximum de pointe \\ \textit{Maximum peak headway}} & \[{f_{peak}}_{max}\] & \[\text{unités}/h\] & \[\frac{60}{{h_{peak}}_{min}}\] & Intervalle maximum de pointe selon contraintes et réglementations nationales, régionales ou locales. \\
+\makecell[r]{Maximum peak headway} & \[{h_{peak}}_{max}\] & \[min\] & \[\frac{60}{{f_{peak}}_{min}}\] & Maximum peak headway according to national, regional, or local constraints and regulations. \\
```
Comment (greenscientist) [inline packages/transition-backend/file/definitions/definitions-en.tex:837]: Semble avoir mixé h et f dnas les équations

File: packages/transition-backend/file/definitions/definitions-en.tex [hunk 16]
```diff
@@ -605,782 +590,771 @@ \subsection*{Temps et durées • \textit{Time and durations}}
...
 \hline
 \label{maximum_off_peak_headway}
-\makecell[r]{Intervalle maximum hors-pointe \\ \textit{Maximum off-peak headway}} & \[{h_{off}}_{max}\] & \[\text{min}\] & \[\frac{60}{{f_{off}}_{min}}\] & Intervalle maximum hors-pointe selon contraintes et réglementations nationales, régionales ou locales. Habituellement valable entre les pointes du matin et de l'après-midi et en soirée. \\
+\makecell[r]{Maximum off-peak headway} & \[{h_{off}}_{max}\] & \[min\] & \[\frac{60}{{f_{off}}_{min}}\] & Maximum off-peak headway according to national, regional, or local constraints and regulations. Usually applicable between morning and afternoon peaks and in the evening. \\
```
Comment (greenscientist) [inline packages/transition-backend/file/definitions/definitions-en.tex:840]: Tu as dropper un \text autour de min

File: packages/transition-backend/file/definitions/definitions-en.tex [hunk 17]
```diff
@@ -605,782 +590,771 @@ \subsection*{Temps et durées • \textit{Time and durations}}
...
-\hline
-\end{longtable}
+    R{.25\NetTableWidth}%
+    C{.06\NetTableWidth}%
+    C{.07\NetTableWidth}%
+    C{.21\NetTableWidth}%
+    L{.41\NetTableWidth}%
+  }
+  \hline
+  \makecell[r]{Definition} & \makecell[c]{Symbol} & \makecell[c]{Unit} & \makecell[c]{Expression} & \makecell[l]{Description} \\\hline
```
Comment (greenscientist) [inline packages/transition-backend/file/definitions/definitions-en.tex:960]: Can you double check this

File: packages/transition-backend/file/definitions/definitions-en.tex [hunk 18]
```diff
@@ -605,782 +590,771 @@ \subsection*{Temps et durées • \textit{Time and durations}}
...
+\makecell[r]{Definition} & \makecell[c]{Symbol} & \makecell[c]{Unit} & \makecell[c]{Expression} & \makecell[l]{Description} \\\hline
 \hline
 \endhead
 \label{capacity}
-\makecell[r]{Capacité \\ \textit{Capacity}} & \[C\] & \[places/h\] & - & Capacité d'une ligne, dans une direction, par heure. \\
+\makecell[r]{Capacity} & \[C\] & \[places/h\] & - & Line capacity, in one direction, per hour. \\
 \hline
 \label{unit_capacity}
-\makecell[r]{Capacité de l'unité \\ \textit{Unit capacity}} & \[c_u\] & \[places\] & - & Capacité d'une unité. \\
+\makecell[r]{Unit capacity} & \[C_{u}\] & \[places\] & \[C_{{u}_{se}} + C_{{u}_{st}}\] & Total number of seated and standing places in the unit. \\
```
Comment (greenscientist) [inline packages/transition-backend/file/definitions/definitions-en.tex:1016]: addtion {} around u in C_u
equation quite different that previous, pleause double check

File: packages/transition-backend/file/definitions/definitions-en.tex [hunk 19]
```diff
@@ -605,782 +590,771 @@ \subsection*{Temps et durées • \textit{Time and durations}}
...
+\makecell[r]{Unit seated capacity} & \[c_{u_{seat}}\] & \[places\] & - & Number of seats in a unit. \\
 \hline
-\label{standee_unit_capacity}
-\makecell[r]{Capacité debout de l'unité \\ \textit{Standee unit capacity}} & \[c_{u_{stand}}\] & \[places\] & - & Capacité de passagers debouts d'une unité. \\
+\label{unit_standees_capacity}
+\makecell[r]{Unit standees capacity} & \[c_{u_{stand}}\] & \[places\] & - & Standing capacity of a unit. \\
 \hline
 \label{vehicle_capacity}
-\makecell[r]{Capacité du véhicule \\ \textit{Vehicle capacity}} & \[c_y\] & \[places\] & - & Capacité d'un véhicule. \\
+\makecell[r]{Vehicle capacity} & \[C_y\] & \[places\] & \[C_{y_{se}} + C_{y_{st}}\] & Number of seated and standing places in the vehicle. \\
```
Comment (greenscientist) [inline packages/transition-backend/file/definitions/definitions-en.tex:1025]: lowercase to upper case c
double check the whole equation

**Review by greenscientist — COMMENTED** [review]
done for now

File: packages/transition-backend/file/definitions/definitions-en.tex (hunk 9 above)
Comment (kaligrafy) [reply packages/transition-backend/file/definitions/definitions-en.tex]: fixed

File: packages/transition-backend/file/definitions/definitions-en.tex (hunk 10 above)
Comment (kaligrafy) [reply packages/transition-backend/file/definitions/definitions-en.tex]: fixed

File: packages/transition-backend/file/definitions/definitions-en.tex (hunk 11 above)
Comment (kaligrafy) [reply packages/transition-backend/file/definitions/definitions-en.tex]: fixed

File: packages/transition-backend/file/definitions/definitions-en.tex (hunk 12 above)
Comment (kaligrafy) [reply packages/transition-backend/file/definitions/definitions-en.tex]: fixed

File: packages/transition-backend/file/definitions/definitions-en.tex (hunk 14 above)
Comment (kaligrafy) [reply packages/transition-backend/file/definitions/definitions-en.tex]: fixed

File: packages/transition-backend/file/definitions/definitions-en.tex (hunk 15 above)
Comment (kaligrafy) [reply packages/transition-backend/file/definitions/definitions-en.tex:837]: L'erreur était en français! corrigé!

File: packages/transition-backend/file/definitions/definitions-en.tex (hunk 16 above)
Comment (kaligrafy) [reply packages/transition-backend/file/definitions/definitions-en.tex:840]: removed in french

File: packages/transition-backend/file/definitions/definitions-en.tex (hunk 13 above)
Comment (kaligrafy) [reply packages/transition-backend/file/definitions/definitions-en.tex]: fixed

File: packages/transition-backend/file/definitions/definitions-en.tex (hunk 8 above)
Comment (kaligrafy) [reply packages/transition-backend/file/definitions/definitions-en.tex]: fixed

File: packages/transition-backend/file/definitions/definitions-en.tex (hunk 17 above)
Comment (kaligrafy) [reply packages/transition-backend/file/definitions/definitions-en.tex]: fixed everywhere, needs double \hline after headers

**Review by greenscientist — APPROVED** [review]
I've looked at the form, not really the content,
